lib/rubygems/errors.rb

```diff
@@ -20,6 +20,41 @@ class LoadError < ::LoadError
     attr_accessor :requirement
   end
 
+  class MissingSpecError < Gem::LoadError
+    def initialize name, requirement
```

Comment:
I personally can't stand the use of unparenthesized method definitions / method calls, fwiw